src/Grid/utils.jl

```diff
@@ -340,3 +340,28 @@ for (func,                   entity_f,            entity_t,     filter_f,
     end
 end
 end
+
+# Hypercube is simply ⨂ᵈⁱᵐ Line :)
+sample_random_point(::Type{Ferrite.RefHypercube{ref_dim}}) where {ref_dim} = Vec{ref_dim}(2.0 .* rand(Vec{ref_dim}) .- 1.0)
+
+# Dirichlet type sampling
+function sample_random_point(::Type{Ferrite.RefSimplex{ref_dim}}) where {ref_dim} 
+    ξ = rand(ref_dim+1)
```

Comment:
To avoid the array allocation
```suggestion
    ξ = ntuple(_ -> rand(), ref_dim+1)
```